src/cookbooks/bigquery.md

```diff
@@ -0,0 +1,148 @@
+# Accessing and Working with BigQuery
+
+This guide will give you a quick introduction to working with data stored
+in [BigQuery](https://cloud.google.com/bigquery/)
+
+As we transition to [GCP](https://cloud.google.com) we will use BigQuery as our primary data warehouse and
+SQL Query engine. BigQuery will eventually replace our previous SQL Query
+Engines, Presto and Athena, and our Parquet data lake.
+## Table of Contents
+<!-- toc -->
+
+# Access
+There are multiple ways to access BigQuery. For most users the primary interface will be [re:dash](https://sql.telemetry.mozilla.org/).
+
+See below for additional interfaces. All other interfaces will require access to be provisioned.
+
+## From re:dash
+All Mozilla users will be able to access BigQuery via [re:dash](https://sql.telemetry.mozilla.org/) through the following Data Sources:
+- `BigQuery Beta`
+- `BigQuery Search Beta`
+    - This group is restricted to users in the re:dash `search` group.
+
+## From `bq` Command-line Tool
+
+- Install the [GCP SDK](https://cloud.google.com/sdk/)
+- Authorize `gcloud` with either your user account or provisioned service account. See documentation [here](https://cloud.google.com/sdk/docs/authorizing).
+    - `gcloud auth login`
+- Set your google project to your team project
+    - `gcloud config set project moz-fx-data-bq-<team-name>`
+    - project name will be provided for you when your account is provisioned.
+
+### `bq` Examples
+List tables in a BigQuery dataset
+``` bash
+bq ls moz-fx-data-derived-datasets:analysis
+```
+Query a table
+ ``` bash
+ bq query --nouse_legacy_sql 'select count(*) from `moz-fx-data-derived-datasets.telemetry.main_summary_v4` where submission_date_s3 = "2019-03-01"'
+ ```
+
+Additional examples and documentation can be found [here](https://cloud.google.com/bigquery/docs/bq-command-line-tool).
+
+## From Databricks
+_Work in progress_
+
+
+## Interfaces
```

Comment:
Logically I think it makes more sense if we should move this section above the "From <tool>" sections, and link to the detailed info for each of the tools